cmd/timestamp-cli/app/verify.go

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
```

Comment:
What do you think about adding an e2e test that calls `timestamp-cli timestamp` and `timestamp-cli verify`, like https://github.com/sigstore/rekor/blob/main/tests/e2e_test.go#L146-L155? We should [tag](https://github.com/sigstore/rekor/blob/main/tests/e2e_test.go#L16) this test so it only runs on request, since it'll require a local instance set up

Feel free to add in another PR, let's just track in the issue

Reply:
That sounds, I'll add it in this PR.

Reply:
Actually, no reason to tag this, we can just have it run against a local instance stood up in the test (Rekor is more complex as it required other dependencies)

You can spin up a local server like here - https://github.com/sigstore/timestamp-authority/pull/96/files#diff-80a202b4fcaa40fcad98b215747d2db42697378ea0b826281a37c53175324a54R135 - and just run `timestamp-cli timestamp` against that